src/HermesWorker/index.js

```diff
@@ -234,6 +235,11 @@ class HermesWorker {
      */
     _onWorkerError(error) {
         console.error(error);
+        if (!this.isLoaded) {
+            this._loadError = error;
+            this._loadedPromise.forEach(({ reject }) => reject(error));
```

Comment:
Could we rename this into `_loadedPromises`? (I know it was not created in this pull request but I find the name without the final 's' a little bit confusing)

Reply:
It's done :stew: :mushroom: :goat: